wsd/TileDesc.hpp

```diff
@@ -181,6 +181,16 @@ class TileDesc final
         return a ^ b;
     }
 
+    TileDesc makeTileForGridPos(int gridX, int gridY) const
```

Comment:
Algorithm reads nicely - thanks =)
This gives me the heebie jeebies though ;-) the TileDesc brings so much important meta-data around, that having new / naked ones being created - and that they might slip into some list/tile-combine etc. worries me somewhat. Could we instead have a throwaway X,Y struct or even parameters we pass down instead of the anchorTile using this ?